src/views/transaction-details/transaction-details.tsx

```diff
@@ -0,0 +1,203 @@
+import { useParams } from "react-router-dom"
```

Comment:
We should reuse the existing component instead of making a new view entirely from scratch. We will add a lot of new stuff to the component and this should be up to date.

Reply:
this has been fixed.  the same component is now being used in both places.